### PR TITLE
Cicero UI -> Web Components

### LINKED ISSUE #301: Cicero UI -> Web Components
# [Cicero UI Reference](https://docs.accordproject.org/docs/ref-cicero-ui.html)

This page will need to be adjusted and possibly expanded to represent the current structure of our UI code. Refer to [these release notes](https://github.com/accordproject/web-components/releases/tag/v0.91.0) as a place to start.

### PR DESCRIPTION
**Pull request by rutvi18**
# Closes #301

### Changes

- <ONE>Added the image in the overview page
- <TWO>Updated the link in started-resources.md

### Screenshots or Video

Screenshot (104)

Screenshot (105)_LI

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [x] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [x] Merging to `master` from `fork:Issue#301`
- [x] Manual accessibility test performed
    - [x] Keyboard-only access, including forms
    - [x] Contrast at least WCAG Level A
    - [x] Appropriate labels, alt text, and instructions

### CODE REVIEW

File: docs/started-resources.md
Comment (Aniruddha-Shriwant) [inline docs/started-resources.md:33]: As the Markdown Editor was moved to `web-components` monorepo you may update that link to https://github.com/accordproject/web-components/tree/master/packages/ui-markdown-editor

File: docs/ref-web-components-overview.md
Comment (Aniruddha-Shriwant) [inline docs/ref-web-components-overview.md:10]: Although I'm not a maintainer I apologize for commenting here but, do we need this image which is of `template-studio-v2` (which is archived now)?
Link of this PR preview build : https://deploy-preview-354--accordproject-old.netlify.app/docs/next/ref-web-components-overview.html

Comment (rutvi18) [reply docs/ref-web-components-overview.md:10]: Yeah..I need to update the image!!

Comment (irmerk) [reply docs/ref-web-components-overview.md]: I think we don't really need an image here at the moment, so this could be removed.

File: docs/started-resources.md
Comment (irmerk) [inline docs/started-resources.md:34]: "Cicero UI" does not exist anymore, so I think we want to remove this reference and expand on what currently exists in `web-components`

**Review by irmerk — CHANGES_REQUESTED** [review]
You'll want to ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit. This will require a rebase, in which you can reword your commit messages to follow the [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)

File: docs/started-resources.md
Comment (irmerk) [inline docs/started-resources.md:34]: Can you make this:

```md
[Contract Editor](https://github.com/accordproject/web-components/tree/master/packages/ui-contract-editor): a rich-text content editor for contract text with embedded clauses.
```

**Review by irmerk — CHANGES_REQUESTED** [review]
Can you remove `docs/assets/reference/sb.png`?

**Comment by rutvi18** [comment]
Screenshot (107)
I am getting this error while signing off. Can u help me understand what can be the reason?

File: docs/started-resources.md
Comment (rutvi18) [reply docs/started-resources.md]: Do you mean to change the content or develop the page?

**Review by Cronus1007 — COMMENTED** [review]
@rutvi18 Here first of all make a git pull and then use the command.It will work fine.

**Comment by Aniruddha-Shriwant** [comment]
Hey @rutvi18, 
I was having the same DCO error before...I had tried this:
`git commit --amend --signoff` and and `git push origin branchName --force` this had worked for me, you can give a try

**Comment by irmerk** [comment]
Since this change is really small, could you rebase the 26 commits down to 1 commit please?

**Comment by jeromesimeon** [comment]
@rutvi18 any update on this?

**Pull request closed without merging**
